coev/evidence_id.go

```diff
@@ -0,0 +1,199 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package coev
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/veraison/corim/comid"
+	"github.com/veraison/corim/encoding"
+	"github.com/veraison/corim/extensions"
+)
+
+// EvidenceID stores an evidence identity. The supported formats as of now are UUID.
```

Comment:
there’s only one supported format

```suggestion
// EvidenceID stores evidence identities. The currently supported format is UUID.
```